server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileRef.java

```diff
@@ -43,17 +45,27 @@
   public static final String PARAM_PROFILE_NAME = "profileName";
   public static final String PARAM_PROFILE_KEY = "profileKey";
 
+  private final OrganizationDto organization;
   private final String key;
   private final String language;
   private final String name;
 
-  private QProfileRef(@Nullable String key, @Nullable String language, @Nullable String name) {
+  private QProfileRef(OrganizationDto organization, @Nullable String key, @Nullable String language, @Nullable String name) {
+    this.organization = organization;
     this.key = key;
     this.language = language;
     this.name = name;
   }
 
   /**
+   * @deprecated provide organization
+   */
+  @Deprecated
+  private QProfileRef(@Nullable String key, @Nullable String language, @Nullable String name) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)